tests/org.jboss.tools.teiid.ui.bot.test/src/org/jboss/tools/teiid/ui/bot/test/PreviewModelTest.java

```diff
@@ -193,7 +189,36 @@ public void customPreview(){
  		assertTrue(checkPreviewCSV(sql,VIEW_ORDERED_TABLE_CSV));
 		assertEquals(227,getCount(sql));
 	}
-	
+
+    @Test
+    public void multipleQueriesTest() {
+        createAndExecuteVDB(VDB_NAME_MULTIPLE, PROJECT_NAME, NAME_ORACLE_MODEL);
+
+        setTextToScrapbookAndExecute(QUERY_MULTIPLE);
+        SQLResult result = DatabaseDevelopmentPerspective.getInstance().getSqlResultsView().getByOperation("Group Execution");
+
+        try {
```

Comment:
this is in fact misuse of try-catch for app logic. If there's a safe check for presence of the shell, you could use sth like:
1. softassert equals
2. check for presence of the error alert, click if present
3. evaluate the softassert so that you fail the test if needed.

If there's not such a test for presence of the dialog, I'd move back to if statement deciding over the result.getStatus variable still, I'd use assertequals rather than assert.fail